ios/RNCPagerScrollView.m

```diff
@@ -0,0 +1,27 @@
+
+#import "RNCPagerScrollView.h"
+
+@implementation RNCPagerScrollView
+- (instancetype)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+    if (self) {
+        self.panGestureRecognizer.delegate = self;
+    }
+    return self;
+}
+
+- (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldRecognizeSimultaneouslyWithGestureRecognizer:(UIGestureRecognizer *)otherGestureRecognizer {
```

Comment:
Also how about moving this delegate to an extension? It will make the code a bit cleaner.

Reply:
I am a bit obj c noob. Do you mean to add a category and make a separate `RNCPagerScrollView+UIGestureRecognizerDelegate.m` file?

Reply:
Let's merge it as is since it's minimal change.